packages/stringext/stringext.1.3.1/descr

```diff
@@ -0,0 +1,5 @@
+Extra string functions for OCaml
+
+Provides a single module named Stringext that provides a grab bug of often used
+but missing string functions from the stdlib. E.g, split, full_split, cut,
+rcut, etc.xtra string functions for OCaml
```

Comment:
Yes. thanks for noticing. Do you mind if I send another PR to fix it after this one is merged. I'd rather just fix the old versions as well all in one go.